nebula-logger/core/main/log-management/lwc/loggerCodeViewer/loggerCodeViewer.js

```diff
@@ -0,0 +1,43 @@
+/*************************************************************************************************
+ * This file is part of the Nebula Logger project, released under the MIT License.               *
+ * See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.   *
+ ************************************************************************************************/
+
+import { LightningElement, api } from 'lwc';
+import { loadScript, loadStyle } from 'lightning/platformResourceLoader';
+import loggerStaticResources from '@salesforce/resourceUrl/LoggerResources';
+
+export default class LoggerCodeViewer extends LightningElement {
+    @api code;
+    @api language;
+    @api startingLineNumber;
+    @api targetLineNumber;
+
+    isLoaded = false;
+
+    _Prism;
+
+    async renderedCallback() {
+        if (this.isLoaded) {
+            return;
+        }
+        await loadStyle(this, loggerStaticResources + '/prism.css');
```

Comment:
should this be consolidated into a `Promise.all` call? or is that overkill with just the two promises

Reply:
I think I tried using `Promise.all` at one point, and something wasn't working quite right at the time.... it was probably due to some other issues in the component, but just to be safe, and since it's just 2 promises right now, I might stick with the 2 separate `await` calls.

Reply:
Nevermind! `Promise.all()` seems to work fine